tests/test_runner.py

```diff
@@ -151,3 +151,55 @@ def test_runner_run_invalidated():
         assert any('Development switches or skip_system_checks were active for this run.' in msg for msg in messages)
     else:
         assert 'Development switches or skip_system_checks were active for this run. This will likely produce skewed measurement data.\n' in messages
+
+def test_runner_with_glob_pattern_filename():
+    """Test that runner works with glob pattern filenames like folder/*.yml"""
+    # Test runner.py with glob pattern that matches multiple files in a folder
+    ps = subprocess.run(
+        ['python3', 'runner.py', '--uri', GMT_DIR, '--filename', 'tests/data/usage_scenarios/runner_filename/*.yml',
+         '--skip-system-checks', '--dev-cache-build', '--dev-no-sleeps', '--dev-no-save'],
+        cwd=GMT_DIR,
+        capture_output=True,
+        text=True,
```

Comment:
we are typically using encoding='utf-8' ... my last knowledge is they are synonyms. If so and text= not mandatory than standardize on encoding=utf-8